slack_sdk/socket_mode/websocket_client/__init__.py

```diff
@@ -180,10 +180,10 @@ def on_close(
 
         self.current_session = websocket.WebSocketApp(
             self.wss_uri,
-            on_open=on_open,  # type: ignore[arg-type]
-            on_message=on_message,  # type: ignore[arg-type]
-            on_error=on_error,  # type: ignore[arg-type]
-            on_close=on_close,  # type: ignore[arg-type]
+            on_open=on_open,
+            on_message=on_message,
+            on_error=on_error,
+            on_close=on_close,
```

Comment:
👁️‍🗨️ note: These are the changes breaking tests at the moment!

🔗 https://github.com/slackapi/python-slack-sdk/actions/runs/18546740363/job/52866164626#step:4:205